projects/dendrites/permutedMNIST/experiments/hyperparameter_search.py

```diff
@@ -0,0 +1,149 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
This file should stay under `experiments` since its a Vernon experiment config file.

Reply:
should be done if I understood correctly what you meant.